TestClient/MainWindow.cs

```diff
@@ -328,17 +329,45 @@ private void btnConnect_Click(object sender, EventArgs e)
 
         private void btnListScenes_Click(object sender, EventArgs e)
         {
-            var scenes = obs.ListScenes();
+            var scenes = obs.GetSceneList().Scenes;
 
             tvScenes.Nodes.Clear();
             foreach (var scene in scenes)
             {
                 var node = new TreeNode(scene.Name);
-                var sources = new List<SceneItemDetails>();
-                sources.AddRange(obs.GetSceneItemList(scene.Name));
-                foreach (var item in sources)
```

Comment:
Why do we need all this in the test client?
Overall groups should be avoided per the Websocket documentation:
"Using groups at all in OBS is discouraged, as they are very broken under the hood."

https://github.com/obsproject/obs-websocket/blob/master/docs/generated/protocol.md#getgroupsceneitemlist

Reply:
Maybe we don't. If I recall correctly, the previous code received and used that info for free from the old API. I was operating under the premise that I would mimic that behavior.